uiowa/uiowa
# Updates to vwu migration

Relates to #2792 





# How to test
Need a local copy of writinguniversity.org in Dev Desktop.
Pull the `vwu` multisite from the Graduate College Acquia Cloud docroot.
Leave the DD Apache/MySQL stack running.

`git checkout writinguniversity-migrate` and `git pull`
`blt ds --site=writinguniversity.org`
`drush @org-writinguniversity.local en writinguniversity_migrate`
`drush @org-writinguniversity.local mim d7_writinguniversity_blog`

Watch that no `array_flip()` warnings appear in the log.

Checkout out the database, and that there no Missing Image tokens were added during migration.
`drush @org-writinguniversity.local sqlc`
`select sourceid1, destid1 from migrate_map_d7_writinguniversity_blog where destid1 in (select entity_id from node__body where body_value like "%<!--Miss%");` will give you the source and destination node IDs for any nodes that have a missing image as result of the migration (or empty set if all went well).

# Note

This PR contains the module/config to run the migration for #2792, but the actual migration and pushing db/files up will need to be run separately, and will still be pending following merge.
